content/courses/connecting-to-offchain-data/oracles.md

```diff
@@ -414,27 +410,34 @@ You can also view the current value stored in an `AggregatorAccountData` account
 client-side in Typescript.
 
 ```typescript
-import { AggregatorAccount, SwitchboardProgram} from '@switchboard-xyz/solana.js'
-
+import { AggregatorAccount, SwitchboardProgram } from "@switchboard-xyz/solana.js";
+import { PublicKey, SystemProgram, Connection } from "@solana/web3.js";
 ...
 ...
-// create keypair for test user
-let user = new anchor.web3.Keypair()
 
-// fetch switchboard devnet program object
+const DEVNET_RPC_URL = "https://api.devnet.solana.com";
+const SOL_USD_SWITCHBOARD_FEED = new PublicKey(
+  "GvDMxPzN1sCj7L26YDK2HnMRXEQmQ2aemov8YBtPS7vR",
+);
+// Create keypair for test user
+let user = new anchor.web3.Keypair();
+
+// Fetch switchboard devnet program object
 switchboardProgram = await SwitchboardProgram.load(
-  "devnet",
-  new anchor.web3.Connection("https://api.devnet.solana.com"),
-  user
-)
+  new Connection(DEVNET_RPC_URL),
+  payer,
+);
 
-// pass switchboard program object and feed pubkey into AggregatorAccount constructor
-aggregatorAccount = new AggregatorAccount(switchboardProgram, solUsedSwitchboardFeed)
+// Pass switchboard program object and feed pubkey into AggregatorAccount constructor
+aggregatorAccount = new AggregatorAccount(
+  switchboardProgram,
+  SOL_USD_SWITCHBOARD_FEED,
+);
 
-// fetch latest SOL price
-const solPrice: Big | null = await aggregatorAccount.fetchLatestValue()
+// Fetch latest SOL price
+const solPrice: Big | null = await aggregatorAccount.fetchLatestValue();
```

Comment:
Is `Big` the same as `BN`? I can't see the import.

Reply:
Yes fixed this by importing - `import { Big } from "@switchboard-xyz/common";`  which is recommended for Oracle data. These are two different `Big` is from the switchboard common library which uses `big.js` and the other is from `bn.js`.

Reply:
OK they're different got it.